### PR TITLE
updates for 22.10

### PR DESCRIPTION
**Pull request by Iroy30**

### CODE REVIEW

File: notebooks/routing/python/notebook_utils/notebook_helpers.py [hunk 1]
```diff
@@ -1,24 +1,5 @@
...
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
-# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
-# DEALINGS IN THE SOFTWARE.
-
+# Copyright (c) 2022, NVIDIA CORPORATION.
```
Comment (ramakrishnap-nv) [inline notebooks/routing/python/notebook_utils/notebook_helpers.py]: MIT Licence is missing

File: notebooks/routing/microservice/notebook_utils/notebook_helpers.py [hunk 2]
```diff
@@ -40,10 +40,11 @@ def gen_plot(df):
         )
     return plt
```
Comment (ramakrishnap-nv) [inline notebooks/routing/microservice/notebook_utils/notebook_helpers.py:61]: Lets add MIT license over here

**Review by rgsl888prabhu — COMMENTED** [review]
Lets remove temp files

**Review by rgsl888prabhu — APPROVED** [review]

**Comment by rgsl888prabhu** [comment]
Don't Merge !!!! Need to create release branch 22.10 and then need to merge this.

**Comment by anandhkb** [comment]
> Don't Merge !!!! Need to create release branch 22.10 and then need to merge this.

Thanks! Ram. Stopped me in my tracks as it should :)

**Comment by anandhkb** [comment]
Kudos and many thanks @Iroy30 and @rgsl888prabhu !!!

**Pull request merged**